Xamarin.Forms.Core/Page.cs

```diff
@@ -146,7 +146,7 @@ public Task<bool> DisplayAlert(string title, string message, string accept, stri
 			if (string.IsNullOrEmpty(cancel))
 				throw new ArgumentNullException("cancel");
 
-			var args = new AlertArguments(title, message, accept, cancel);
+			var args = new AlertArguments(title, message, accept, cancel, Visual);
```

Comment:
I don't think this is good here, visual should be a "global" thing, not per request. This also lends itself to having different popups on different pages - which I believe is a very bad thing.

Reply:
The code here looks to have a reference to the sender page so you can just send that to PresentAlert opposed to adding it onto the arguments

https://github.com/xamarin/Xamarin.Forms/blob/b1241170b61b9cef6c0963c5b5725d0e7bf443d8/Xamarin.Forms.Platform.iOS/Platform.cs#L41

Reply:
That is probably a way better thing to do!